src/filesystem/impls/filer/lib/handlers.js

```diff
@@ -3,10 +3,11 @@
 define(function (require, exports, module) {
     "use strict";
 
-    var Content = require("filesystem/impls/filer/lib/content");
-    var HTMLRewriter = require("filesystem/impls/filer/lib/HTMLRewriter");
-    var CSSRewriter = require("filesystem/impls/filer/lib/CSSRewriter");
-    var Path  = require("filesystem/impls/filer/BracketsFiler").Path;
```

Comment:
Don't change things like this, it messes with git blame history without adding value (i.e. you're not actually changing the code here, just the `var` style).